kitsune/search/v2/documents.py

```diff
@@ -15,30 +19,41 @@
 
 
 class WikiDocument(SumoDocument):
-    url = field.Keyword()
-    updated = field.Date()
+    """ES Document for Knowledge Base.
+
+    Each document in ES is built on top of a parent KB article.
+    All the children are included in the same document.
+    """
 
+    updated = field.Date()
     product_ids = field.Keyword(multi=True)
     topic_ids = field.Keyword(multi=True)
+    locale = field.Keyword()
+    category = field.Keyword()
+    recent_helpful_votes = field.Integer()
 
     # Document specific fields (locale aware)
+    url = SumoLocaleAwareKeywordField(store=True)
+    slug = SumoLocaleAwareKeywordField(store=True)
```

Comment:
Do we need the slug since we have the url?

Reply:
slug != url b/c the url contains also the locale. I guess it depends on the usage. We can surely construct one from the other. Slug allows another way to extract the document. I don't have a good answer here. It can go both ways